custom/airport/airportcore/resources/airportcore-items.xml

```diff
@@ -31,158 +36,272 @@
+					<modifiers optional="false" />
 				</attribute>
-				<attribute qualifier="timeArr" type="java.lang.Integer">
+				<attribute qualifier="goods" type="java.lang.Integer">
 					<persistence type="property"></persistence>
-					<modifiers optional="true"/>
+					<modifiers optional="false" />
 				</attribute>
 			</attributes>
 		</itemtype>
-		
+
+		<itemtype code="Passenger" extends="User" generate="true"
+			autocreate="true">
+			<deployment table="passenger" typecode="20107" />
+			<attributes>
+				<attribute qualifier="fiscalCode" type="java.lang.String">
+					<persistence type="property"></persistence>
+					<modifiers optional="false" />
+				</attribute>
+				<attribute qualifier="birthday" type="java.util.Date">
+					<persistence type="property"></persistence>
+					<modifiers optional="true" />
+				</attribute>
+				<attribute qualifier="sex" type="Gender">
+					<persistence type="property"></persistence>
+					<modifiers optional="false" />
+				</attribute>
+				<attribute qualifier="nationality" type="java.lang.String">
+					<persistence type="property"></persistence>
+					<modifiers optional="false" />
+				</attribute>
+			</attributes>
+		</itemtype>
+
+
 		<itemtype code="Route" generate="true" autocreate="true">
-			<deployment table="route" typecode="21348" />
+			<deployment table="route" typecode="20103" />
 			<attributes>
 				<attribute qualifier="codeRoute" type="java.lang.String">
-						<persistence type="property"></persistence>
-						<modifiers optional="false"/>
+					<persistence type="property"></persistence>
+					<modifiers optional="false" />
 				</attribute>
-				<attribute qualifier="flight" type="Flight">
-						<persistence type="property"></persistence>
-						<modifiers optional="true"/>
+				<attribute qualifier="airportDep" type="Airport">
+					<persistence type="property"></persistence>
+					<modifiers optional="true" />
 				</attribute>
-				<attribute qualifier="dateRouteDep" type="java.util.Date">
-						<persistence type="property"></persistence>
-						<modifiers optional="true"/>
+				<attribute qualifier="airportArr" type="Airport">
+					<persistence type="property"></persistence>
+					<modifiers optional="true" />
 				</attribute>
-				<attribute qualifier="dateRouteArr" type="java.util.Date">
-						<persistence type="property"></persistence>
-						<modifiers optional="true"/>
+				<attribute qualifier="routeTime" type="java.lang.Integer">
+					<persistence type="property"></persistence>
+					<modifiers optional="false" />
 				</attribute>
 			</attributes>
 		</itemtype>
-		
-		
-		<itemtype code="Passenger" extends="User" generate="true" autocreate="true">
-			<deployment table="passenger" typecode="23000" />
+
+		<itemtype code="Booking" generate="true" autocreate="true">
+			<deployment table="booking" typecode="20108" />
 			<attributes>
-				<attribute qualifier="codeFiscal" type="java.lang.String">
+				<attribute qualifier="sit" type="java.lang.Integer">
```

Comment:
il posto in un aereo di solito è rappresentato da una lettera e un numero, quindi credo sia meglio una stringa